app/views/hackathons/digest_mailer/digest.html.erb

```diff
@@ -1,13 +1,29 @@
 <p>
   Check out
-  <% if (count = @digest.listings.count) > 1 %>
-    these <%= count %> hackathons
+  <% if (count = @digest.listings.count { |listing| listing.previously_listed_for_recipient? }) > 1 %>
+    these <%= count %> newly listed hackathons
   <% else %>
-    this hackathon
+    this newly listed hackathon
   <% end %>
   coming up near you.
 </p>
 
 <% @listings_by_subscription.each do |subscription, listings| %>
```

Comment:
I would say maybe set these in the mailer so you don't have to repeat the `reject`s? Like `@new_listings_by_subscription` and `@previous_listings_by_subscription`